frontend/redux/sagas.js

```diff
@@ -36,11 +38,22 @@ export function* requestWorkload() {
   }
 }
 
+export function* completeWorkload() {
+  try {
+    const workloadToSend = yield select(selectWorkload);
+    const workloadPostResponse = yield call(postWorkload, workloadToSend);
+    yield put(workloadPostResponse);
```

Comment:
I agree, but I'm thinking of that as a new feature. The workload post api does not yet return a new workload.